css/css-text/i18n/zh/css-text-line-break-zh-cpm-loose.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html  lang="en" >
+<head>
+<meta charset="utf-8"/>
+<title>CSS text, linebreaks: centred punctuation (loose,zh)</title>
+<link rel="author" title="Richard Ishida" href="mailto:ishida@w3.org">
+<link rel="help" href="https://drafts.csswg.org/css-text-3/#line-break-property">
+<script src="../../../../resources/testharness.js"></script>
```

Comment:
@xfq I agree with @gsnedders, and apologise that I should have mentioned that earlier. However, please make these changes only in the files submitted to WPT, because top-level absolute paths don't work well when the tests are used in the i18n test suite.